package/firmware/cypress-nvram/Makefile

```diff
@@ -53,14 +53,14 @@ $(eval $(call BuildPackage,cypress-nvram-43430-sdio-rpi-3b))
 define Package/cypress-nvram-43430-sdio-rpi-zero-w
   $(Package/cypress-nvram-default)
   TITLE:=CYW43430 NVRAM for Raspberry Pi Zero W
-  DEPENDS:=@TARGET_bcm27xx
+  DEPENDS:=@TARGET_bcm27xx +cypress-nvram-43430-sdio-rpi-3b
   CONFLICTS:=brcmfmac-firmware-43430-sdio-rpi-zero-w
 endef
 
 define Package/cypress-nvram-43430-sdio-rpi-zero-w/install
 	$(INSTALL_DIR) $(1)/lib/firmware/brcm
-	$(INSTALL_DATA) \
-		$(PKG_BUILD_DIR)/brcmfmac43430-sdio.raspberrypi,model-zero-w.txt \
+	$(LN) \
+		brcmfmac43430-sdio.raspberrypi,3-model-b.txt \
```

Comment:
I would prefer if you merge all the nvram files at least for one wifi card together into one package. You could create a `cypress-nvram-43430` and add there all nvram files and all symlinks for 43430. They are pretty small and only text files. the ipkg should still be small.

Reply:
@hauke @kuanyili I don't like this approach, since we can end up with a considerable amount of files for those devices with the most common SoCs, but I can live with that...

Reply:
@Noltari @hauke Those text files really are quite small. The combined text file size in `brcm` of `linux-firmware` is less than 36 kB. If we count only those for 43430, less than 4 kB; for 43455, less than 10 kB. And it will be even smaller after compression (SquashFS).

That's why I also pulled NVRAM files for other devices in while performing NVRAM package consolidation. This could benefit the support for many devices, including some RPi Zero 2 W boards. 

I'm fine with both though. If the decision is to keep NVRAM packages separate by device, just strip the last two commits away, and we are good to go.

Reply:
@kuanyili I've just merged the first 6 commits, excluding the last two ones until we agree on what to do:
https://github.com/openwrt/openwrt/compare/ebe2b7190b7d8815a588eaf8a5cfdf9edfd85c36...a110418027a4c6f97c33980b8e4d7f07d39726de

Thanks!